cinema/serializers.py

```diff
@@ -125,10 +121,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
In the `validate` method, you're calling `super().validate(attrs=attrs)`. As you're not modifying the `attrs` in any way before passing them to the parent's `validate` method, you can simplify this call to `super().validate(attrs)`. This will make your code cleaner and easier to read.